oviewer/convert_align.go

```diff
@@ -185,17 +189,17 @@ func appendSpaces(lc contents, num int) contents {
 }
 
 func (a *align) isShrink(col int) bool {
-	if len(a.shrink) <= col {
+	if len(a.columnAttrs) <= col {
 		return false
 	}
-	return a.shrink[col]
+	return a.columnAttrs[col].shrink
 }
 
 func (a *align) isRightAlign(col int) bool {
-	if len(a.rightAlign) <= col {
+	if len(a.columnAttrs) <= col {
 		return false
 	}
-	return a.rightAlign[col]
+	return a.columnAttrs[col].rightAlign
```

Comment:
col starts at 0?

I'm asking because len starts at 1 if there is an element

I'm unsure the code does what is expected

What about this?

```go
 func (a *align) isRightAlign(col int) bool { 

         if c, found := a.columnAttrs[col]; found { 
                  return c.rightAlign
          } 
         return false
  }
```

Reply:
yes. col starts from 0.
Need to determine the range because of slice.